src/styles/blocks/minute-axis.scss

```diff
@@ -0,0 +1,30 @@
+.minute-axis {
+  height: 15px;
+  width: $watch-size * 0.5;
+  display: flex;
+  justify-content: flex-end;
+  align-self: center;
+  position: absolute;
+
+  &__hand {
+    height: 15px;
+    width: $watch-size * 0.25;
+    background-color: red;
+  }
+
+  animation-duration: 3600s;
+  animation-timing-function: linear;
```

Comment:
Anyway those values, except `animation-duration` are the same as in `second-axis.scss`, you can use a `@mixin` or `@extend` for this to not duplicate code. Example:

Create class e.g name`.hand-animation`, and you can use this class name in HTML `class="second-axis__hand hand-animation"` and `class="minute-axis__hand hand-animation"`

Or you can try to use them here after `animation-duration` as `@extend .hand-animation`, same below in `second-axis.scss` file :)